item/serializers.py

```diff
@@ -0,0 +1,148 @@
+from rest_framework import serializers
+from item.validators import positive_price_validator
+from item.models import Currency, Item, Price
+
+
+class CurrencyDeleteSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Currency
+        fields = ('id', )
+        read_only_field = ('id',)
+
+
+class CurrencyListSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Currency
+        fields = ('name', 'code')
+        read_only_field = ('id',)
```

Comment:
I'm not sure but it will be good if you'll specify all `read_only_fields` explicitly in fields
And also read_only_field**s** not read_only_field

Reply:
Change it everywhere ctrl+shift+r will help you